src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -57,4 +61,19 @@ public void toModelType_duplicateConcerts_throwsIllegalValueException() throws E
                 JsonSerializableAddressBook.MESSAGE_DUPLICATE_CONCERT, dataFromFile::toModelType);
     }
 
+    @Test
+    public void toModelType_duplicateConcertContact_throwsIllegalValueException() throws Exception {
+        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(DUPLICATE_CONCERT_COONTACT_FILE,
+                JsonSerializableAddressBook.class).get();
+        assertThrows(IllegalValueException.class,
+                JsonSerializableAddressBook.MESSAGE_DUPLICATE_CONCERT_CONTACT, dataFromFile::toModelType);
+    }
+
+    @Test
+    public void toModelType_invalidConcertContactFile_throwsIllegalValueException() throws Exception {
```

Comment:
Might be better to split the test into the existing one for invalidConcertContactConcert, and another one for invalidConcertContactPerson, for more comprehensive equivalence partitioning.

Reply:
Would also need to split this file:
src/test/data/JsonSerializableAddressBookTest/invalidConcertContactAddressBook.json